Templates/TimerTrigger/function.json

```diff
@@ -2,7 +2,7 @@
   "disabled": false,
   "bindings": [
     {
-      "name": "myTimer",
+      "name": "timerinfo",
```

Comment:
this should match the defaultParameterName in binding.json
 "defaultParameterName": "myTimer",